app/screens/About/AboutScreen.tsx

```diff
@@ -1,9 +1,13 @@
 import React from "react";
 import { ScrollView, View } from "react-native";
+// eslint-disable-next-line import/no-unresolved
```

Comment:
why do we suddently need this line?

Reply:
Fixed it so eslint also reads the tsconfig.json file now. This also required me to change a lot of "incorrect" default imports